### ISSUE TITLE
Deprecated: Return type of Symfony\Component\Process\Process::getIterator

### ISSUE THREAD

**Issue by pjcvijay** [KnpLabs/snappy#457]
I'm getting below error in PHP 8.1:

`( ! ) Deprecated: Return type of Symfony\Component\Process\Process::getIterator($flags = 0) should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /var/www/autouonline/vendor/symfony/process/Process.php on line 567`

I'm using v1.4.1, looks like it's not compatible with PHP 8.0.

Any suggestions please?

Thanks!

**Comment by garak** [comment]
This library is indeed compatible with PHP 8.0 (and even 8.1 by the way).
The deprecation you get is raised by an external library.

**Comment by PedroTroller** [comment]
@pjcvijay Indeed, according to the error you encounter, it is the version of symfony/process that triggers this warning, snappy is compatible with the latest version of the component, can you try to update symfony/process?
